modules/developer-cli-odo-installing-odo-on-windows.adoc

```diff
@@ -6,29 +6,23 @@
 
 = Installing {odo-title} on Windows
 
-== Binary installation
-
-. Download the latest link:https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-windows-amd64.exe[`odo.exe`] file.
-. Add the location of your `odo.exe` to your `GOPATH/bin` directory.
-
-[discrete]
-== Setting the `PATH` variable for Windows 7/8
-
-The following example demonstrates how to set up a path variable. Your binaries can be located in any location, but this example uses `C:\go-bin` as the location.
-
-. Create a folder at `C:\go-bin`.
-. Right click *Start* and click *Control Panel*.
-. Select *System and Security* and then click *System*.
-. From the menu on the left, select the *Advanced systems settings* and click *Environment Variables* at the bottom.
-. Select *Path* from the *Variable* section and click *Edit*.
-. Click *New* and type `C:\go-bin` into the field or click *Browse* and select the directory, and click *OK*.
-
-[discrete]
-== Setting the `PATH` variable for Windows 10
-
-Edit `Environment Variables` using search:
-
-. Click *Search* and type `env` or `environment`.
-. Select *Edit environment variables for your account*.
-. Select *Path* from the *Variable* section and click *Edit*.
-. Click *New* and type `C:\go-bin` into the field or click *Browse* and select the directory, and click *OK*.
+The `{odo-title}` CLI for windows is available to download as a binary and as an archive.
```

Comment:
See all my comments on `Installing odo on Linux` and `Installing odo on MacOS` and apply them here to `Installing odo on Windows`.

Reply:
I have made the changes across all OSes - so they should now be consistent and updated appropriately